styles/shadows.css

```
@@ -0,0 +1,6 @@
+@theme {
```

Stylelint (17.5.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)